src/Analyzers/CSharp/Tests/AddParameter/AddParameterTests.cs

```diff
@@ -3340,4 +3340,110 @@ void M(string s)
             }
             """);
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/71428")]
+    public async Task TestAddConstructorParameterWithExistingField_PrimaryConstructor()
+    {
+        await TestInRegularAndScript1Async(
```

Comment:
all the below behaviors we get for free by piggy backing on the "initialize parameter from primary constructor" code that is now moved to shared helpers.